yoga/node/Node.h

```diff
@@ -144,6 +146,24 @@ class YG_EXPORT Node : public ::YGNode {
     return children_.size();
   }
 
+  const LayoutableChildren getLayoutChildren() const {
```

Comment:
The other children already do this, so makes sense to keep with the pattern (no change needed/requested), but it is bizarre that the methods which return mutable references to children are marked const.